app/views/About.js

```diff
@@ -11,16 +11,17 @@ import {
   Text,
 } from 'react-native';
 
+import TracingStrategyContext from '../TracingStrategyContext';
 import packageJson from '../../package.json';
 import fontFamily from './../constants/fonts';
 import { NavigationBarWrapper, Typography } from '../components';
-import { useAssets } from '../TracingStrategyAssets';
 
 import { Colors, Spacing } from '../styles';
 
 export const AboutScreen = ({ navigation }) => {
   const { t } = useTranslation();
-  const { aboutHeader } = useAssets();
+  const { useCopy } = useContext(TracingStrategyContext);
```

Comment:
this we could probably consolidate into one custom hook: something like: `useStrategyContent`